packages/delivery-service/migrations/20240229170017_/migration.sql

```diff
@@ -0,0 +1,36 @@
+-- CreateTable
+CREATE TABLE "EncryptedMessage" (
```

Comment:
Isn't that supposed to be part of the BE. If the prisma schema has moved there wen don't need to keep the migration in the delivery-service dir

Reply:
absolutely: fab74eba9a84c4d87707db43e4745d51c0c87a66